### ISSUE TITLE
Clean the Forum

### ISSUE THREAD

**Issue by Aryan14** [romdi/IOS-Assets#16]
Clean the whole forum and start over
You can always save those old skool archives somewhere.
But IOS really needs a new start.

**Comment by Walsheh** [comment]
(Y)

**Comment by ghost** [comment]
Yeah Rom's mentioned it before, and I'm not against it, but I think it's best to wait until we've got stuff to integrate it with and wait until we're nearer a full release. I'll just install a fresh forum, the old one will stay as is.
